uisdk/src/test/java/tech/dojo/pay/uisdk/domain/IsSDKInitializedCorrectlyUseCaseTest.kt

```diff
@@ -0,0 +1,69 @@
+package tech.dojo.pay.uisdk.domain
+
+import org.junit.Assert.assertEquals
+import org.junit.Test
+import org.mockito.kotlin.mock
+import org.mockito.kotlin.whenever
+import tech.dojo.pay.uisdk.domain.entities.PaymentIntentDomainEntity
+import tech.dojo.pay.uisdk.entities.DojoPaymentType
+
+class IsSDKInitializedCorrectlyUseCaseTest {
+
+    @Test
+    fun `given calling isSDKInitiatedCorrectly with Virtual Terminal PaymentIntent, isSDKInitiatedCorrectly then should return true`() {
+        // Given
+        val paymentIntent: PaymentIntentDomainEntity = mock()
+        val paymentType = DojoPaymentType.VIRTUAL_TERMINAL
+        whenever(paymentIntent.isVirtualTerminalPayment).thenReturn(true)
+        val useCase = IsSDKInitializedCorrectlyUseCase()
```

Comment:
Why are you initializing all the time the use case for each test? you can do this one time or do it on @Before